setup.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 
+. ./scripts/genkeys.sh --source-only
+
 echo "Initializing .env file..."
 
 if test ! -f ".env"; then
   cp .env.defaults .env
+  genkeys .env
```

Comment:
Moving to a separate file feels like an overkill, especially when there's no reason to use this file outside of this location.

```suggestion
    placeHolder="{GenKey()}"
    for str in $(grep ${placeHolder} $1); do sed -i "s/${placeHolder}/$(openssl rand -hex 12)/" $1; done
```